ingestion/src/metadata/profiler/api/models.py

```diff
@@ -113,3 +119,15 @@ class ProfilerResponse(ConfigModel):
     def __str__(self):
         """Return the table name being processed"""
         return f"Table [{self.table.name.__root__}]"
+
+
+class ThreadPoolMetrics(ConfigModel):
+    """thread pool metric"""
```

Comment:
what's a threadpool metric? 😅

Reply:
They are just the pool of metrics that we submit for computation on the profiler side